Assets/_BForBoss/_Utility/Editor/PropertyDrawers/ResolverPropertyDrawer.cs

```diff
@@ -0,0 +1,369 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using PerigonGames;
+using UnityEditor;
+using UnityEditor.IMGUI.Controls;
+using UnityEngine;
+using UnityEngine.Rendering.VirtualTexturing;
+using Object = System.Object;
+
+namespace BForBoss
+{
+    [CustomPropertyDrawer(typeof(ResolveAttribute))]
+    public class ResolverPropertyDrawer : PropertyDrawer
+    {
+        private const float RESOLVER_WIDTH = 80f;
+        private const float OBJECT_FIELD_BUTTON_WIDTH = 18f;
+        private const float RESOLVER_BUTTON_WIDTH = 20f;
+        private const float ENUMERABLE_BOX_OFFSET = 25f;
+        private const float STRUCT_OFFSET = 15f;
+
+        private bool _isContentInitialized = false;
+        private bool _includeInactiveGameObjects;
+        
+        private bool _isList = false;
+        private bool _isArray = false;
+        private bool _isDeclaredInStruct = false;
+        
+        //Button Content
+        private GUIContent _childResolveContent;
+        // private GUIContent _sceneResolveContent;
+        private GUIContent _parentResolveContent;
+        // private GUIContent _siblingResolveContent;
+        private GUIContent _selfResolveContent;
+        private GUIContent _clearContent;
+        
+        //Button Rects
+        private Rect _childResolveRect;
+        // private Rect _sceneResolveRect;
+        private Rect _parentResolveRect;
+        // private Rect _siblingResolveRect;
+        private Rect _selfResolveRect;
+        private Rect _clearContentRect;
+
+        private GUIStyle _clearContentGUIStyle;
+        
+        private enum ResolveType
+        {
+            FromChildren,
+            FromParent,
+            FromSelf
+        }
+        
+        
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            if (!IsResolvable(property))
+            {
+                EditorGUI.PropertyField(position, property, label);
+                return;
+            }
+
+            if (!_isContentInitialized)
+            {
+                InitializeContent();
+
+                ResolveAttribute resolveAttributeAttribute = (ResolveAttribute) attribute;
+                _includeInactiveGameObjects = resolveAttributeAttribute.IncludeInactive;
```

Comment:
How do you include Inactive currently? are we able to change it?

Reply:
If I'm reading it right there's an optional constructor and it defaults to true, so if you don't want to include inactive children you'd go `[Resolve(false)]`

Reply:
Yup exactly, it will default to search inactive gameobjects. and you can toggle that off with the optional parameter

Reply:
How difficult is it to implement it like the Odin style? 
`[Resolve(IncludeInactive = false)]`

Reply:
Wait im still confused, what is the difference between how it is now ,
![image](https://user-images.githubusercontent.com/23490604/147861561-e41bb918-c22e-406b-93a7-7efd14c2d707.png),

And how you want it to be?

Reply:
I didn't see it. Ita fine as it is then